source/lac/dynamic_sparsity_pattern.cc

```diff
@@ -407,6 +408,34 @@ DynamicSparsityPattern::symmetrize ()
 
 
 
+template <typename SparsityPatternTypeLeft, typename SparsityPatternTypeRight>
+void
+DynamicSparsityPattern::compute_mmult_pattern (
+  const SparsityPatternTypeLeft &left,
+  const SparsityPatternTypeRight &right)
+{
+  Assert (left.n_cols() == right.n_rows(),
+          ExcDimensionMismatch(left.n_cols(), right.n_rows()));
+
+  this->reinit(left.n_rows(),right.n_cols());
+
+  typename SparsityPatternTypeLeft::iterator
+  it_left = left.begin(),
+  end_left = left.end();
+  for (; it_left!=end_left; ++it_left)
+    {
+      const unsigned int m = it_left->column();
```

Comment:
I have read that Europeans use (n,m) to index into matrices, whereas Americans use (m,n). Indeed, when I read `m`, I think of a row, not a columns. Maybe rename this variable to `j` (using the uncontroversial (i,j))?